napari/_qt/_qapp_model/_tests/utils.py

```diff
@@ -0,0 +1,25 @@
+import qtpy
+from qtpy.QtWidgets import QMenu
+
+
+def get_submenu_action(window_menu, submenu_text, action_text):
```

Comment:
Sorry last thing, the function name is pretty self explanatory but could we add a docstring? We can mention that `findAction` cannot find submenu actions.

Not sure if this functionality is something Talley is interested in supporting, we'll see what he says (https://github.com/pyapp-kit/app-model/issues/147#issuecomment-2164428076)